server/parsec/backend/utils.py

```diff
@@ -26,8 +27,9 @@
     from parsec.backend.client_context import BaseClientContext
 
     Ctx = TypeVar("Ctx", bound=BaseClientContext)
-    T = TypeVar("T")
-    P = ParamSpec("P")
+
+T = TypeVar("T")
+P = ParamSpec("P")
```

Comment:
This does not seem related to the license change, was it the formatter? Is the indentation change wanted?